model/src/main/scala/fr/iscpif/schelling/quantity/initial/Container.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2015 Romain Reuillon
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package fr.iscpif.schelling.quantity.initial
+
+import fr.iscpif.schelling.quantity._
+
+import scala.util.Random
+
+trait Container {
+
+  def size : Int
+
+  var values : Seq[Seq[Cell]]
```

Comment:
This should be the return of the function container, not a shared mutable value.